src/styles/main.scss

```diff
@@ -0,0 +1,87 @@
+$stopwatch-size: 500px;
+$hand-width: 10px;
+$small-hand-height: $stopwatch-size / 3;
+$center: $stopwatch-size / 2;
+
+//a minute is arranged for 6s since it is boring to watch
+$animation-minute-time: 6s;
+
+@mixin watchsize {
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+}
+
+body {
+  margin: 0;
+}
+
+.page__container {
+  margin: 50px auto;
+}
+
+.container {
+  position: relative;
+
+  @include watchsize();
+
+  border-radius: 50%;
```

Comment:
You can include the border-radius property into the `watchsize` mixin to save space.

Reply:
good catch